src/simulators/matrix_product_state/matrix_product_state.hpp

```diff
@@ -489,8 +497,30 @@ void State::add_metadata(ExperimentResult &result) const {
   result.metadata.add(
     MPS::get_sample_measure_alg(),
     "matrix_product_state_sample_measure_algorithm");
+  MPS::clear_log();
 } 
 
+void State::add_reporting_metadata(ExperimentResult &result) const {
+  result.metadata.add("{" + MPS::output_log() + "}", "MPS_log_data");
+}
+
+void State::output_bond_dimensions(const Operations::Op &op) const {
+  MPS::print_to_log("I", instruction_number, ":", op.name, " on qubits ", op.qubits[0]);
+  for (uint_t index=1; index<op.qubits.size(); index++) {
+    MPS::print_to_log(",", op.qubits[index]);
+  }
+  MPS::print_to_log(", BD= [");
```

Comment:
The spacing could be nicer, currently it looks like this:
```
I0:unitary on qubits 0,4, BD= [2 1 1 1 1 1 1 1 1]`
```
You could insert a space between "I0:" and "unitary", or between "BD" and "=", or alternatively for consistency remove the space between "BD=" and "[".

Reply:
ok, I prefer to reduce the spaces to make this printout more compact.